server/courselist.rb

```diff
@@ -620,10 +638,12 @@ def self.getURLUniqname(url)
 
   ############ check out the esb course call and time the performance.
   check_esb = lambda do
+
+    config_hash = settings.latte_config
     st = Stopwatch.new("timing of check url")
     st.start
 
-    course_data = dataProviderCourse(@@check_uniqname, @@check_termid.to_s)
+    course_data = dataProviderCourse(settings.latte_config[:check_uniqname], settings.latte_config[:check_termid].to_s)
```

Comment:
should this be config_hash[:check_uniqname] and config_hash[:check_termid] instead?